src/pages/api/teams/[teamId]/digests/[digestId]/index.ts

```diff
@@ -46,19 +46,21 @@ router
         Here is a list of document titles sorted from most recent to oldest, separared by ; signs : ${lastDigestTitles.join(
           ';'
         )} 
-        Just guess the next document title. Don't add any other sentence in your response.
+        Just guess the next document title. Don't add any other sentence in your response. If you can't guess a logical title, just write idk.
         `;
 
           try {
             const response = await openAiCompletion({ prompt });
             const guessedTitle = response[0]?.message?.content;
-
-            await client.team.update({
-              where: { id: digest?.teamId },
-              data: {
-                nextSuggestedDigestTitle: guessedTitle,
-              },
-            });
+            const canPredict = guessedTitle !== 'idk';
+            if (canPredict) {
+              await client.team.update({
```

Comment:
I think we can probably remove the `await` here since it doesn't impact the digest response.